{{cookiecutter.project_slug}}/{{cookiecutter.project_slug}}/users/views.py

```diff
@@ -27,6 +29,11 @@ def get_success_url(self):
     def get_object(self):
         return User.objects.get(username=self.request.user.username)
 
+    def form_valid(self, form):
+        form.save()
+        messages.add_message(self.request, messages.INFO, _("Infos succefully updated"))
+        return super(UserUpdateView, self).form_valid(form)
```

Comment:
We are running Python 3.6+, we don't need to pass the class and instance to `super()`:

```suggestion
        return super().form_valid(form)
```